.eslintrc

```diff
@@ -3,15 +3,16 @@
     "react-app",
     "airbnb",
     "plugin:jsx-a11y/recommended",
-    "prettier",
-    "prettier/react",
     "plugin:import/errors",
     "plugin:import/warnings",
     "plugin:import/typescript"
   ],
-  "plugins": ["jsx-a11y", "prettier"],
+  "plugins": ["jsx-a11y"],
   "rules": {
-    "semi": 0,
```

Comment:
I think we should keep `"semi": 0,` in or? If not, we'll get ton of complains about missing semicolons everywhere I think.

Reply:
for me it didn't change anything one way or the other, so 🤷 
happy to keep it in though